core/src/main/scala/org/apache/spark/serializer/SerializerManager.scala

```diff
@@ -70,6 +70,8 @@ private[spark] class SerializerManager(
   private[this] val compressRdds = conf.getBoolean("spark.rdd.compress", false)
   // Whether to compress shuffle output temporarily spilled to disk
   private[this] val compressShuffleSpill = conf.getBoolean("spark.shuffle.spill.compress", true)
+  // Size of the chunks to be used in the ChunkedByteBuffer
+  private[this] val chunkSizeMb = conf.getSizeAsMb("spark.memory.chunkSize", "4m").toInt
```

Comment:
The name `spark.memory.chunkSize` looks too generic.
How about `spark.memory.serializer.chunkSize` or others?

Reply:
I renamed it to spark.memory.serializer.chunkSize